apps/webapp/next.config.js

```diff
@@ -1,18 +1,32 @@
-module.exports = {
-  reactStrictMode: true,
-  transpilePackages: ['ui'],
-  images: {
-    domains: ['avatar.vercel.sh'],
-  },
-  webpack: config => {
-    config.module.rules.push({
-      test: /\.mp4$/,
-      type: 'asset/resource',
-      generator: {
-        filename: 'videos/[hash][ext][query]',
-      },
-    });
+module.exports = () => {
+  const isDev = process.env.NODE_ENV !== 'production';
+  const prodConfig = !isDev ? { output: 'export' } : {};
 
-    return config;
-  },
+  return {
+    ...prodConfig,
+    reactStrictMode: true,
+    transpilePackages: ['ui'],
+    images: {
+      unoptimized: true,
```

Comment:
Why unoptimized?

Reply:
without this, image from domain (avatar.vercel.sh) doesnt show

Reply:
This does not seem right. We don't want to hurt all of our images just for that one. Think this requires more investigation on how to get that avatar to show correctly. 

For example, you could do:

```typescript
import Image from 'next/image'
 
const UnoptimizedImage = (props) => {
  return <Image {...props} unoptimized />
}
```

if that avatar is giving problems.

Reply:
I'll add this as a separate component, in case you need it in the future